### DISCUSSION TITLE
Adding a hidden ssid to the network list via webgui.

### DISCUSSION THREAD

**Discussion by mp035** [RaspAP/raspap-webgui#802] [General]
Hi,
I have a need to connect the wifi client to a hidden SSID.  Is there currently a mechanism in the webgui to add a hidden SSID (ie one that cannot be seen on a scan)?

**Comment by billz** [comment]
Not at the moment. IIRC it has been requested before, so would make a nice addition.

**Reply by mp035** [reply]
I'll be adding that feature to my personal repository.  Are you interested in a PR?
